packages/editor/src/client/blocks/mapLeafletBlock.tsx

```diff
@@ -22,148 +25,76 @@ Icon.Default.mergeOptions({
-          }
-        }}>
-        <TileLayer
-          url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-          attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
-        />
-        <Marker position={[centerLat, centerLng]}></Marker>
-      </Map>
-      <Box marginTop={Spacing.ExtraSmall}>
-        <TextInput
-          label="Caption"
-          description="Map description"
-          value={caption}
-          onChange={event => {
-            onChange({...value, caption: event.target.value})
-          }}
-          marginBottom={Spacing.Small}
-        />
-      </Box>
-    </>
-  )
-}
-// This function will be used for Map-items
-export function MapLeafletItemElement({value, onChange}: FieldProps<MapLeafletItem>) {
-  const [isChooseModalOpen, setChooseModalOpen] = useState(false)
-  const [isEditModalOpen, setEditModalOpen] = useState(false)
-
-  //@ts-ignore
-  const {image, title, lat, lng, description} = value
-
-  return (
-    <>
-      <Box display="flex" flexDirection="row">
-        <Card
-          overflow="hidden"
-          width={200}
-          height={150}
-          marginRight={Spacing.ExtraSmall}
-          flexShrink={0}>
-          <PlaceholderInput onAddClick={() => setChooseModalOpen(true)}>
-            {image && (
-              <Box position="relative" width="100%" height="100%">
-                <Box position="absolute" zIndex={ZIndex.Default} right={0} top={0}>
-                  <IconButton
-                    icon={MaterialIconImageOutlined}
-                    title="Choose Image"
-                    margin={Spacing.ExtraSmall}
-                    onClick={() => setChooseModalOpen(true)}
-                  />
-                  <IconButton
-                    icon={MaterialIconEditOutlined}
-                    title="Edit Image"
-                    margin={Spacing.ExtraSmall}
-                    onClick={() => setEditModalOpen(true)}
-                  />
-                  <IconButton
-                    icon={MaterialIconClose}
-                    title="Remove Image"
-                    margin={Spacing.ExtraSmall}
-                    onClick={() => onChange(value => ({...value, image: null}))}
-                  />
-                </Box>
-                {image.previewURL && <Image src={image.previewURL} width="100%" height="100%" />}
-              </Box>
-            )}
-          </PlaceholderInput>
-        </Card>
-        <Box flexGrow={1}>
-          <TypographicTextArea
-            variant="h1"
-            placeholder="Title"
-            value={title}
-            onChange={e => {
-              const title = e.target.value
-              onChange(value => ({...value, title}))
+      <Box position="relative" width="100%" height="100%">
+        <Map
+          center={[centerLat, centerLng]}
+          zoom={zoom}
+          style={{width: '100%', height: '45vh'}}
```

Comment:
why this height?

Reply:
The default height (if I don't set anything) of the map-leaflet is rather narrow and not very convenient for the user. I found that 45vh is a good compromise between having enough space but not filling out too much of the screen. I have chosen vh since it adapts to the viewport of the user's screen and is thus independent of size and resolution.